include/otdoa_al/otdoa_api.h

```diff
@@ -0,0 +1,404 @@
+/*
+ * Copyright (c) 2025 PHY Wireless, Inc.
+ *
+ * SPDX-License-Identifier: LicenseRef-PHYW
+ */
+
+#ifndef PHYWI_OTDOA_API__
+#define PHYWI_OTDOA_API__
+
+#include <stdint.h>
+#include <stddef.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file otdoa_api.h
+ *
+ * @defgroup phywi_otdoa_api hellaPHY OTDOA module
+ * @{
+ * @brief API Definitions for the hellaPHY OTDOA module.
+ */
+
+#define OTDOA_API_MAX_CELLS	  75
+#define OTDOA_API_MAX_SESSION_LEN 250
+
+#ifdef HOST
+#define LOG2_COMPRESS_WINDOW                                                                       \
+	15 /* use large decomp. window on HOST so we can decompress files compressed with gzip */
+#else
+#define LOG2_COMPRESS_WINDOW                                                                       \
+	9 /* Log2 of compression window size (i.e. 10=>1024B compression window) */
+#endif
+
+/**
+ * @brief Maximum length of the uBSA file path
+ */
+#define OTDOA_API_MAX_UBSA_FILE_PATH 50
+
+typedef enum {
```

Comment:
Coding Style: We don't use typedefs in Zephyr. I guess these should be removed from OTDOA AL API as this is not something where we are stuck with an external binary.